lnd_responses/rpc_channel_update_as_update.js

```diff
@@ -20,6 +20,8 @@ const now = () => new Date().toISOString();
       disabled: <Forwarding is Disabled Bool>
       fee_base_msat: <Forwarding Base Fee Millitokens String>
       fee_rate_milli_msat: <Forwarding Fee Rate Per Million String>
+      inbound_fee_base_msat: <Inbound Forwarding Base Fee Millitokens Number>
```

Comment:
should be discount?

Reply:
This is the incoming object from lnd, don't think they mention `discount`